File: src/data/Model/Cms360/AcademicCalendar/AcademicSectionMap.cs
/*
*   Auther: H.Muhammad Kamran
*   email: hmuhdkamran@gmail.com
*   contact: +92 (313 / 333) 9112 845
*/

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Cms360.Data.Model
{
    [Table("AcademicSectionLink", Schema = "AcademicCalendar")]
    public partial class AcademicSectionMap
    {
		[Key]
		[Required]
		public Guid AcademicSectionLinkId { get; set; }
		public Guid AcademicCalendarMasterId { get; set; }
		public Guid SectionCourseLinkId { get; set; }
		public int StatusId { get; set; }

    }

	public partial class AcademicSectionMapVW

    {

        [Key]

        [Required]

        public Guid AcademicSectionLinkId { get; set; }

        public Guid AcademicCalendarMasterId { get; set; }

        public Guid SectionCourseLinkId { get; set; }

        public Guid CampusProgramId { get; set; }

        public string AcademicMaster { get; set; }

        public string SectionName { get; set; }

        public Guid SubCityId { get; set; }

        public Guid SessionId { get; set; }

        public Guid ClassId { get; set; }

        public Guid BoardId { get; set; }

        public int StatusId { get; set; }



    }


}

File: src/ui/app/root/application/models/academiccalendar/academicsectionmap.ts
export interface IAcademicSectionMap {
	academicSectionLinkId: string;
	academicCalendarMasterId: string;
	sectionCourseLinkId: string;
	statusId: number;
}

export interface IAcademicSectionMapVW {
	academicSectionLinkId: string;
	academicCalendarMasterId: string;
	sectionCourseLinkId: string;
	academicMaster: string;
	sectionName: string;
	subCityId: string;
	classId: string;
	boardId: string;
	statusId: number;
}